### PR TITLE
Add onomy

### PR DESCRIPTION
**Pull request by Parth4337**
Adding Onomy Data

### CODE REVIEW

File: onomy/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      ],
+      "base": "anom",
+      "name": "Nom",
+      "display": "nom",
+      "symbol": "NOM",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/onomy/images/nom.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/onomy/images/nom.svg"
+      },
+      "coingecko_id": "onomy-protocol",
```
Comment (JeremyParish69) [inline onomy/assetlist.json:25]: I don't see this in CoinGecko's list. https://api.coingecko.com/api/v3/coins/list
can we please remove until it's been added?

File: onomy/chain.json [hunk 2]
```diff
@@ -0,0 +1,116 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "onomy",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://onomy.io/",
+  "pretty_name": "Onomy",
+  "chain_id": "onomy-mainnet-1",
+  "bech32_prefix": "onomy",
```
Comment (JeremyParish69) [inline onomy/chain.json:9]: Please register this prefix to SLIP-0173:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: onomy/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+          "exponent": 18
+        }
+      ],
+      "base": "anom",
+      "name": "Nom",
+      "display": "nom",
+      "symbol": "NOM",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/onomy/images/nom.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/onomy/images/nom.svg"
```
Comment (JeremyParish69) [inline onomy/assetlist.json:23]: It looks to me that the svg version of the image doesn't have transparency in the corners. Could you please find one that does?

File: onomy/chain.json (hunk 2 above)
Comment (cgdusek) [reply onomy/chain.json:9]: Submitted PR

File: onomy/assetlist.json (hunk 3 above)
Comment (cgdusek) [reply onomy/assetlist.json]: Removed

File: onomy/chain.json (hunk 2 above)
Comment (cgdusek) [reply onomy/chain.json:9]: https://github.com/satoshilabs/slips/pull/1477

File: onomy/assetlist.json (hunk 1 above)
Comment (cgdusek) [reply onomy/assetlist.json]: Removed coingecko_id

File: onomy/chain.json [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+  "daemon_name": "onomyd",
+  "node_home": "$HOME/.onomyd",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
```
Comment (JeremyParish69) [inline onomy/chain.json:17]: do we have fee range token data? e.g.,
  "fixed_min_gas_price": 0,
  "low_gas_price": 0,
  "average_gas_price": 0.025,
  "high_gas_price": 0.04
Osmosis will need this anyway.

File: onomy/chain.json [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+    {
+      "kind": "mintscan",
+      "url": "https://www.mintscan.io/onomy-protocol",
+      "tx_page": "https://www.mintscan.io/onomy-protocol/txs/${txHash}",
+      "account_page": "https://www.mintscan.io/onomy-protocol/account/${accountAddress}"
+    }
+  ],
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/onomy/images/nom.png",
+    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/onomy/images/nom.svg"
```
[the remark this answers is not in the document]
Comment (Parth4337) [reply onomy/chain.json]: Updated

File: onomy/chain.json (hunk 4 above)
Comment (Parth4337) [reply onomy/chain.json:17]: Updated

**Pull request merged**